lib/stacker/cli.rb

```diff
@@ -111,18 +115,6 @@ def dump stack_name = nil
       end
     end
 
-    desc "fmt [STACK_NAME]", "Re-format template JSON"
-    def fmt stack_name = nil
```

Comment:
I actually use this. Are we removing support for JSON? I prefer it over yml for templates, since that's how AWS stores it, and having a standard formatting for the JSON eliminates argument—a la. go fmt.